src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -545,11 +545,21 @@ protected function initializeContainer()
                         return $previousHandler ? $previousHandler($type, $message, $file, $line) : false;
                     }
 
+                    $backtrace = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS);
```

Comment:
I'm suggesting we don't need this trace at all: it's always the same so it provides no value.